numpy/__init__.pxd

```diff
@@ -585,7 +585,7 @@ cdef extern from "numpy/arrayobject.h":
     npy_bool PyArray_CheckStrides (int, int, npy_intp, npy_intp, npy_intp *, npy_intp *)
     dtype PyArray_DescrNewByteorder (dtype, char)
     object PyArray_IterAllButAxis (object, int *)
-    #object PyArray_CheckFromAny (object, dtype, int, int, int, object)
+    #object PyArray_CheckFromAny (object, dtype, int, int, int, int, object)
```

Comment:
Unfortunately we can't add arguments to existing API functions, that breaks users.

Reply:
Fixed